packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -47,9 +46,6 @@ const CheckoutPage = props => {
 
     const classes = mergeClasses(defaultClasses, propClasses);
 
-    const windowSize = useWindowSize();
-    const isMobile = windowSize.innerWidth <= 960;
```

Comment:
Spoke with @jimbo we may actually wnat to use window size now considering upcoming UX changes.

Reply:
Reverted this to use the hook again :)